sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -105,15 +105,15 @@ def _to_generated_list(targets):
         return [target._to_generated() for target in targets]  # pylint: disable=protected-access
 
 
-class BatchDocumentInput(object):  # pylint: disable=useless-object-inheritance
+class DocumentTranslationInput(object):  # pylint: disable=useless-object-inheritance
     # pylint: disable=C0301
     """Definition for the input batch translation request.
 
     :param source_url: Required. Location of the folder / container or single file with your
      documents.
     :type source_url: str
     :param targets: Required. Location of the destination for the output.
-    :type targets: list[StorageTarget]
+    :type targets: list[TranslationTarget]
     :keyword str source_language: Language code
```

Comment:
I suppose we should also rename `source_language` to `source_language_code` to be consistent.

Reply:
done